### DISCUSSION TITLE
Better organize the repository

### DISCUSSION THREAD

**Discussion by sebastianjnuwu** [Acode-Foundation/Acode#906] [General]
Hi @deadlyjack, It would be cool and very important for you, a few collaborators with me and other devs to tidy up the look of the repository, but how? The issues page has many resolved issues and another one out of context that would be good close currently has 180 open problems but in fact it was supposed to have +/- 30

**Comment by bajrangCoder** [comment]
Yeah, It would be cool to move Acode repository to a separate `Acode` Organisation and there contributor have permission to interact with issues etc(like closing , giving labels etc)

**Comment by sebastianjnuwu** [comment]
@deadlyjack Happy 2024 to you 🥂

Think about the idea, it would be very interesting to do this, you don't have to worry about reading problems that are often the same, as well as organizing better to be able to work and evolve
